3주차/11568/11568_python_제로.py

```diff
@@ -0,0 +1,26 @@
+import copy
+import sys
+from collections import defaultdict
+from bisect import bisect_left
+
+n = int(sys.stdin.readline())
+lst = list(map(int, sys.stdin.readline().split(" ")))
```

Comment:
```suggestion
readline = sys.stdin.readline
n = int(readline())
lst = list(map(int, readline().split()))
```
입력함수가 반복되는 경우위와 같이 중복처리하면 편할 거 같아요!!
그리고 `split()`은 디폴트가 공백 기준이니 `" "`가 없어도 되어 보여요